### ISSUE TITLE
Кол-во покупок с 5.10.2015

### ISSUE THREAD

**Issue by vampig** [Maxx53/SCMBot#471]
поделитесь инфой всяк сюда входящий. у всех покупки упали с 5го октября? или особо без изменений?

**Comment by Justmau5** [comment]
Упали, почти до нуля.

**Comment by kajo6** [comment]
agreed, purchase almost zero, i think the last steam update kill the scm bot

**Comment by uto4ka** [comment]
С 5-ого числа, все совсем плохо стало. Есть идеи что случилось?
Кошачий бог приди.

**Comment by vzapas** [comment]
Maxx53, бот уже 2 дня сканирует вхолостую, ни одной покупки при включенном скане html, хотя в ночь с 12 на 13 октября на этом же итеме и таких же настройках бота все было норм. Перезагрузки не помогли, бубен с различными настройками в качестве включенного Json также не помогли. Заметил только то, что при скане html проскакивает =
![default](https://cloud.githubusercontent.com/assets/15053065/10513311/ce79d738-7357-11e5-80fd-4ade53d7d50e.JPG)
Кошачий бог приди, помоги!!!

**Comment by vzapas** [comment]
И кстати, работа вкладки "новые поступления" перестала правильно работать 
![1](https://cloud.githubusercontent.com/assets/15053065/10513673/338d88b6-735a-11e5-99b6-3e8a5781e55a.JPG) Справа - страница предмета; слева - новые поступления

**Comment by Maxx53** [comment]
всем, кто грешит на сломанный процесс покупки, экспериментируем - добавляем убогую дешевую шмотку, выставляем завышенную желаемую цену, отмечаем флаг "купить", 1 штуку, запускаем скан.

**Comment by vzapas** [comment]
Maxx53, покупка работает! Проверял утром, перед тем как написать данные сообщения. Как я считаю, скан проскальзывает покупку шмотки по демпинговой цене  т.е. бот перестал видеть минимальную цену, примерно также ее не видно на  графиках стима. 
![default](https://cloud.githubusercontent.com/assets/15053065/10520317/3e9b865a-7372-11e5-9d42-a1e7dad5c220.jpg), К примру я выставлял на покупку шмотки 3.9 руб и покупал , сейчас идет проскальзывание какой-то =) Надеюсь, мысль смог донести

**Comment by Rocky54** [comment]
@vzapas мысль я твою не до конца понял. Но суть остаётся та же. Кол-во покупок стало меньше, и доход тоже

**Comment by rnd0m** [comment]
https://github.com/vzapas, на графике показывают не минимальную цену, а среднюю. Если я тоже конечно тебя понял хоть сколь-нибудь

**Comment by Maxx53** [comment]
https://github.com/Maxx53/SCMBot/issues/475
